juno_magic/extensions/wamp.py

```diff
@@ -243,23 +302,32 @@ def __init__(self, shell):
 
         self._parser = self.generate_parser()
 
-    def set_connection(self, wamp_connection):
-        log.msg("SET_CONNECTION: {}".format(wamp_connection))
-        # Make sure things get cleaned up no matter why the reset happens
-        try:
-            self._wamp.leave()
-            self._wamp_runner.cancel()
-            del self._wamp
-        except (CancelledError, AttributeError) as e:
-            log.msg(e)
-            pass
+    if _ENABLE_START_BRIDGE:
```

Comment:
went over w/ @pramukta , gonna add one change to brdige.py and then i think we're good. The dispatching works great w/ jupyter-react